src/tutorial/microbe_stage/MicrobeTutorialGUI.cs

```diff
@@ -81,7 +81,7 @@ public bool MicrobeWelcomeVisible
             }
             else
             {
-                microbeWelcomeMessage.Visible = false;
+                microbeWelcomeMessage.ClosePopup();
```

Comment:
Why don't these work with the old approach anymore?
Even if the Visible = false was modified to call this.
Anyway if there's no way to make that modification, then what can be we do to reduce the risk that someone accidentally uses `Visible = false`?